python_modules/libraries/dagster-tableau/dagster_tableau/translator.py

```diff
@@ -318,9 +319,21 @@ def get_dashboard_spec(self, data: TableauTranslatorData) -> AssetSpec:
                     project_id=workbook_data.properties["projectLuid"],
                 )
             },
+            kinds = {"tableau", "dashboard"}
         )
 
     def get_data_source_spec(self, data: TableauTranslatorData) -> AssetSpec:
+        kinds = {"tableau"}
+        if data.properties["hasExtracts"] is True:
+            kinds.add("extract")
+        else:
+            kinds.add("live")
+
+        if data.properties["isPublished"] is True:
+            kinds.add("published")
+        else:
+            kinds.add("embedded")
+        
```

Comment:
nit: to ease readability

```suggestion
        kinds = {
            [
                "tableau", 
                *["extract" if data.properties["hasExtracts"] else "live"], 
                *["published" if data.properties["isPublished"] else "embedded"], 
            ]
        }
```

Reply:
also adapted tests